src/components/EpisodeBlock.tsx

```diff
@@ -27,36 +27,38 @@ const EpisodeBlock: React.FC<EpisodeBlockProps> = (props) => {
   };
 
   return (
-    <div className="episode-block">
-      <div className={classnamesHeader}>
-        {!isFirst && !isLast && (
-          <button className="episode-arrow episode-up" onClick={() => onClick(Direction.Up)}>
-            🔼
-          </button>
-        )}
-        {isLast && <div className="episode-locked">🔒</div>}
-        <>{title}</>
-        {!isOneBeforeLast && !isLast && (
-          <button className="episode-arrow episode-down" onClick={() => onClick(Direction.Down)}>
-            🔽
-          </button>
-        )}
-      </div>
-      <div className="episode-content">
-        <div className="episode-writers">
-          <b>Writer{writersSuffix}:</b> {writers.join(', ')}
+    <div className={`episode-block`}>
+      <div className={`episode-ribbon-number1-index${index} episode-ribbon bg-color-${color}` }/>
+      <div className={`episode-ribbon-number2-index${index} episode-ribbon bg-color-${color}`}/>
+      <div className={`episode-ribbon-number3-index${index} episode-ribbon bg-color-${color}`}/>
+      <div className={`episode-content`}>
+        <div className={`episode-header`}>
+          {!isFirst && !isLast ? (
+            <button className="episode-arrow episode-up" onClick={() => onClick(Direction.Up)}>
+              🔼
+            </button>
+          ) : <div></div>}
+          {isLast && <div className="episode-locked">🔒</div>}
+          <>{title}</>
+          {!isOneBeforeLast && !isLast ? (
+            <button className="episode-arrow episode-down" onClick={() => onClick(Direction.Down)}>
+              🔽
+            </button>) : <div></div>
+          }
         </div>
-        {director && (
-          <div className="episode-director">
-            <b>Director:</b> {director}
+        <div className={`episode-content-separator bg-color-${color}`}></div>
+        <div className="episode-details">
+          <div className="episode-writers">
+            <b>Writer{writersSuffix}:</b> {writers.join(', ')}
+          </div>
+          {director && (
+            <div className="episode-director">
+              <b>Director:</b> {director}
+            </div>
+          )}
+          <div className="episode-number">
+            <b>Episode #{defaultNumber}</b>
           </div>
-        )}
-
-        <div className="episode-color">
-          <b>Color:</b> {getDotEmoji(color)} {color === Color.Pink ? `brown (instead of ${color})` : color}
```

Comment:
Let's have a gradient from pink to brown for the episode line and for the background. The limitation about having to use brown comes from the fact square/dot emojis exist for a limited set of colors which include purple that I use for violet episode, but there's no pink square/dot emoji. So I picked brown. Only remaining one would have been black but I read that the introduction to the show will be called Black.